CLN (0.12.1) install fails due to missing python module 'mako'

Branch: `dev`
CLN version: `0.12.1`
Python: `3.9.2-3`


Steps to reproduce:
Run `config.scripts/cl.install.sh install`



Result:
```
- installDependencies()
Reading package lists... Done
Building dependency tree... Done
Reading state information... Done
autoconf is already the newest version (2.69-14).
automake is already the newest version (1:1.16.3-2).
build-essential is already the newest version (12.9).
gettext is already the newest version (0.21-4).
git is already the newest version (1:2.30.2-1).
libgmp-dev is already the newest version (2:6.2.1+dfsg-1+deb11u1).
libsodium-dev is already the newest version (1.0.18-1).
libtool is already the newest version (2.4.6-15).
net-tools is already the newest version (1.60+git20181103.0eebece-1).
python3 is already the newest version (3.9.2-3).
libsqlite3-dev is already the newest version (3.34.1-3).
zlib1g-dev is already the newest version (1:1.2.11.dfsg-2+deb11u2).
0 upgraded, 0 newly installed, 0 to remove and 1 not upgraded.
Reading package lists... Done
Building dependency tree... Done
Reading state information... Done
libpq-dev is already the newest version (13.8-0+deb11u1).
postgresql is already the newest version (13+225).
0 upgraded, 0 newly installed, 0 to remove and 1 not upgraded.
Requirement already satisfied: pip in /usr/local/lib/python3.9/dist-packages (22.3.1)
WARNING: Running pip as the 'root' user can result in broken permissions and conflicting behaviour with the system package manager. It is recommended to use a virtual environment instead: https://pip.pypa.io/warnings/venv
Requirement already satisfied: poetry in /home/bitcoin/.local/lib/python3.9/site-packages (1.2.2)
Requirement already satisfied: keyring>=21.2.0 in /home/bitcoin/.local/lib/python3.9/site-packages (from poetry) (23.11.0)
Requirement already satisfied: crashtest<0.4.0,>=0.3.0 in /home/bitcoin/.local/lib/python3.9/site-packages (from poetry) (0.3.1)
Requirement already satisfied: importlib-metadata<5.0,>=4.4 in /home/bitcoin/.local/lib/python3.9/site-packages (from poetry) (4.13.0)
Requirement already satisfied: html5lib<2.0,>=1.0 in /usr/local/lib/python3.9/dist-packages (from poetry) (1.1)
Requirement already satisfied: urllib3<2.0.0,>=1.26.0 in /home/bitcoin/.local/lib/python3.9/site-packages (from poetry) (1.26.12)
Requirement already satisfied: poetry-plugin-export<2.0.0,>=1.1.2 in /home/bitcoin/.local/lib/python3.9/site-packages (from poetry) (1.2.0)
Requirement already satisfied: platformdirs<3.0.0,>=2.5.2 in /home/bitcoin/.local/lib/python3.9/site-packages (from poetry) (2.5.3)
Requirement already satisfied: dulwich<0.21.0,>=0.20.46 in /home/bitcoin/.local/lib/python3.9/site-packages (from poetry) (0.20.50)
Requirement already satisfied: cachy<0.4.0,>=0.3.0 in /home/bitcoin/.local/lib/python3.9/site-packages (from poetry) (0.3.0)
Requirement already satisfied: pexpect<5.0.0,>=4.7.0 in /home/bitcoin/.local/lib/python3.9/site-packages (from poetry) (4.8.0)
Requirement already satisfied: pkginfo<2.0,>=1.5 in /home/bitcoin/.local/lib/python3.9/site-packages (from poetry) (1.8.3)
Requirement already satisfied: tomlkit!=0.11.2,!=0.11.3,<1.0.0,>=0.11.1 in /home/bitcoin/.local/lib/python3.9/site-packages (from poetry) (0.11.6)
Requirement already satisfied: poetry-core==1.3.2 in /home/bitcoin/.local/lib/python3.9/site-packages (from poetry) (1.3.2)
Requirement already satisfied: cleo<2.0.0,>=1.0.0a5 in /home/bitcoin/.local/lib/python3.9/site-packages (from poetry) (1.0.0a5)
Requirement already satisfied: cachecontrol[filecache]<0.13.0,>=0.12.9 in /home/bitcoin/.local/lib/python3.9/site-packages (from poetry) (0.12.11)
Requirement already satisfied: packaging>=20.4 in /usr/local/lib/python3.9/dist-packages (from poetry) (21.3)
Requirement already satisfied: jsonschema<5.0.0,>=4.10.0 in /home/bitcoin/.local/lib/python3.9/site-packages (from poetry) (4.17.0)
Requirement already satisfied: shellingham<2.0,>=1.5 in /home/bitcoin/.local/lib/python3.9/site-packages (from poetry) (1.5.0)
Requirement already satisfied: requests-toolbelt<0.10.0,>=0.9.1 in /home/bitcoin/.local/lib/python3.9/site-packages (from poetry) (0.9.1)
Requirement already satisfied: requests<3.0,>=2.18 in /home/bitcoin/.local/lib/python3.9/site-packages (from poetry) (2.28.1)
Requirement already satisfied: virtualenv!=20.4.5,!=20.4.6,>=20.4.3 in /home/bitcoin/.local/lib/python3.9/site-packages (from poetry) (20.16.6)
Requirement already satisfied: msgpack>=0.5.2 in /home/bitcoin/.local/lib/python3.9/site-packages (from cachecontrol[filecache]<0.13.0,>=0.12.9->poetry) (1.0.4)
Requirement already satisfied: lockfile>=0.9 in /home/bitcoin/.local/lib/python3.9/site-packages (from cachecontrol[filecache]<0.13.0,>=0.12.9->poetry) (0.12.2)
Requirement already satisfied: pylev<2.0.0,>=1.3.0 in /home/bitcoin/.local/lib/python3.9/site-packages (from cleo<2.0.0,>=1.0.0a5->poetry) (1.4.0)
Requirement already satisfied: six>=1.9 in /usr/lib/python3/dist-packages (from html5lib<2.0,>=1.0->poetry) (1.16.0)
Requirement already satisfied: webencodings in /usr/local/lib/python3.9/dist-packages (from html5lib<2.0,>=1.0->poetry) (0.5.1)
Requirement already satisfied: zipp>=0.5 in /home/bitcoin/.local/lib/python3.9/site-packages (from importlib-metadata<5.0,>=4.4->poetry) (3.10.0)
Requirement already satisfied: attrs>=17.4.0 in /home/bitcoin/.local/lib/python3.9/site-packages (from jsonschema<5.0.0,>=4.10.0->poetry) (22.1.0)
Requirement already satisfied: pyrsistent!=0.17.0,!=0.17.1,!=0.17.2,>=0.14.0 in /home/bitcoin/.local/lib/python3.9/site-packages (from jsonschema<5.0.0,>=4.10.0->poetry) (0.19.2)
Requirement already satisfied: SecretStorage>=3.2 in /home/bitcoin/.local/lib/python3.9/site-packages (from keyring>=21.2.0->poetry) (3.3.3)
Requirement already satisfied: jeepney>=0.4.2 in /home/bitcoin/.local/lib/python3.9/site-packages (from keyring>=21.2.0->poetry) (0.8.0)
Requirement already satisfied: jaraco.classes in /home/bitcoin/.local/lib/python3.9/site-packages (from keyring>=21.2.0->poetry) (3.2.3)
Requirement already satisfied: pyparsing!=3.0.5,>=2.0.2 in /usr/local/lib/python3.9/dist-packages (from packaging>=20.4->poetry) (3.0.9)
Requirement already satisfied: ptyprocess>=0.5 in /home/bitcoin/.local/lib/python3.9/site-packages (from pexpect<5.0.0,>=4.7.0->poetry) (0.7.0)
Requirement already satisfied: charset-normalizer<3,>=2 in /home/bitcoin/.local/lib/python3.9/site-packages (from requests<3.0,>=2.18->poetry) (2.1.1)
Requirement already satisfied: certifi>=2017.4.17 in /usr/lib/python3/dist-packages (from requests<3.0,>=2.18->poetry) (2020.6.20)
Requirement already satisfied: idna<4,>=2.5 in /usr/local/lib/python3.9/dist-packages (from requests<3.0,>=2.18->poetry) (2.8)
Requirement already satisfied: filelock<4,>=3.4.1 in /home/bitcoin/.local/lib/python3.9/site-packages (from virtualenv!=20.4.5,!=20.4.6,>=20.4.3->poetry) (3.8.0)
Requirement already satisfied: distlib<1,>=0.3.6 in /home/bitcoin/.local/lib/python3.9/site-packages (from virtualenv!=20.4.5,!=20.4.6,>=20.4.3->poetry) (0.3.6)
Requirement already satisfied: cryptography>=2.0 in /home/bitcoin/.local/lib/python3.9/site-packages (from SecretStorage>=3.2->keyring>=21.2.0->poetry) (38.0.3)
Requirement already satisfied: more-itertools in /home/bitcoin/.local/lib/python3.9/site-packages (from jaraco.classes->keyring>=21.2.0->poetry) (9.0.0)
Requirement already satisfied: cffi>=1.12 in /home/bitcoin/.local/lib/python3.9/site-packages (from cryptography>=2.0->SecretStorage>=3.2->keyring>=21.2.0->poetry) (1.15.1)
Requirement already satisfied: pycparser in /home/bitcoin/.local/lib/python3.9/site-packages (from cffi>=1.12->cryptography>=2.0->SecretStorage>=3.2->keyring>=21.2.0->poetry) (2.21)
PATH="$HOME/.local/bin:$PATH"
Installing dependencies from lock file

...
wiregen common/status_wiregen.h
Traceback (most recent call last):
  File "/home/bitcoin/lightning/tools/generate-wire.py", line 27, in <module>
    from mako.template import Template
ModuleNotFoundError: No module named 'mako'
make: *** [Makefile:331: common/status_wiregen.h] Error 1
```

--- a/build_sdcard.sh
+++ b/build_sdcard.sh
@@ -320,7 +320,7 @@ sudo update-alternatives --install /usr/bin/pip pip /usr/bin/pip3 1
 # pytesseract mechanize PySocks urwid Pillow requests
 # 3. Nyx
 # setuptools
-python_libs="grpcio==1.42.0 googleapis-common-protos==1.53.0 toml==0.10.2 j2cli==0.3.10 requests[socks]==2.21.0 protobuf==3.20.1"
+python_libs="grpcio==1.42.0 googleapis-common-protos==1.53.0 toml==0.10.2 j2cli==0.3.10 requests[socks]==2.21.0 protobuf==3.20.1 mako==1.2.3"
 torbox_libs="pytesseract mechanize PySocks urwid Pillow requests setuptools"
 sudo -H python3 -m pip install --upgrade pip
 sudo -H python3 -m pip install ${python_libs} ${torbox_libs}

